config/initializers/elastic_search.rb

```diff
@@ -0,0 +1,16 @@
+
+require 'elasticsearch'
+
+module Vocabulary
+  module Elasticsearch
+    client_opts = Settings.elasticsearch
+    # this is required to allow updating the adapter used by faraday
+    # the argument is a symbol to the client so we need to change it
+    # to make sure it works -- really required for testing using fake web
+    client_opts[:adapter] = client_opts['adapter']
+    @@client = ::Elasticsearch::Client.new client_opts
```

Comment:
Pretty minor nit, but I'm not a fan of @@ variables. Maybe singleton could be used here: http://ruby-doc.org/stdlib-2.3.3/libdoc/singleton/rdoc/Singleton.html